sigs.yaml

```diff
@@ -1194,6 +1194,18 @@ sigs:
       tz: PT (Pacific Time)
       frequency: weekly
       url: https://docs.google.com/document/d/1oLXv5_rM4f645jlXym_Vd7AUq7x6DV-O87E6tcW1sjU/edit
+    - description: Office Hours European Edition
```

Comment:
could we make it clear in the description that this is for end users and not q&a for upstream? thanks~

Reply:
Would something like `Community Office Hours European Edition` sound better?

Thoughts @castrojo ?